README.md

```diff
@@ -35,6 +35,42 @@ schema path usage.
     psql hhs-acf-uc-api_test
     set search_path=uacportal,uac_health
 
+## Queries and usage
+
+This application exposes and handles in memory no PII. All PII stays in
+the existing database extracting only calculations, dates and ids and
+other non-sensitive information.
+
+Only the queries needed for the dashboard have been created:
+
+#### Referrals on a given date
+
+    /referrals # defaults to today
```

Comment:
Hi @baccigalupi ... I'm here because @mgwalker asked me to help out with code review. Everything looks great to me, just adding some small comments/feedback.

I think there should be documentation for the HTTP verbs for these methods. They seem like (and are) read (GET) methods, but it would be nice to have that explicit in the docs.